### PR TITLE
Move light as child of camera

### PR DESCRIPTION
**Pull request by mkrus**
Will move along with it and not affect viewAll

### CODE REVIEW

**Comment by wheeland** [comment]
Will this not break lighting in case a glTF camera is used?

**Comment by wheeland** [comment]
Yup, just checked. With a glTF camera, the spotlight will not be visible anymore.

File: tools/assetpipelineeditor/main.qml [hunk 1]
```diff
@@ -41,21 +41,6 @@ Kuesa.SceneEntity {
     readonly property Layer texturePreviewLayer: Layer {}
     readonly property Layer materialPreviewLayer: Layer {}
 
-    // let this point light wander around with the camera to create some shiny lighting
-    Entity {
-        id: pointLightEntity
-        property Camera followedCamera: frameGraph.camera
-        parent: followedCamera.parent
```
Comment (lemirep) [inline tools/assetpipelineeditor/main.qml]: I think the correct fix is actually parent: followedCamera no ?

Comment (wheeland) [reply tools/assetpipelineeditor/main.qml]: You are right. I forgot that QCamera is just a regular entity with a Transform-component.

**Comment by wheeland** [comment]
Fixed here: https://github.com/KDAB/kuesa/pull/82

**Pull request closed without merging**